src/Sauce/Sausage/SauceConfig.php

```diff
@@ -75,8 +83,8 @@ public static function GetBuild()
         return false;
     }
 
-    public static function WriteConfig($username, $access_key) {
-        file_put_contents(CONFIG_PATH, "{$username},{$access_key}");
+    public static function WriteConfig($username, $access_key, $sauce_host = 'saucelabs.com') {
+        file_put_contents(CONFIG_PATH, "{$username},{$access_key},$sauce_host");
```

Comment:
any reason not to follow the same pattern as the other variables?

Reply:
I tried to keep the variable pattern as the others.
Regulary I use camelCase, instead, so I might have missed some?

Reply:
I mean that the others have braces around them and the new one doesn't. Is there a reason?

Reply:
Ah, this is inconsistent, I will change that!